ogles_gpgpu/CMakeLists.txt

```diff
@@ -56,50 +61,36 @@ else()
     # have to be made option in order to support this.  For now the
     # OSMESA option is excluded from those platforms in the CMake control
     # logic.
-    
-    find_library(OSMESA_LIBRARY OSMesa)
-    if(NOT OSMESA_LIBRARY)
-      message(FATAL_ERROR "OGLES_GPGPU was configured to use OSMesa, but the library could not be found")
-    endif()
 
-    get_filename_component(OSMESA_PATH ${OSMESA_LIBRARY} DIRECTORY)
+    find_package(osmesa REQUIRED)
+    find_package(glapi REQUIRED)
+    find_package(egl REQUIRED)
 
-    find_library(GLAPI_LIBRARY glapi HINTS ${OSMESA_PATH} PATHS ${OSMESA_PATH})
```

Comment:
We need to make sure find_package() replacements use the `${OSMESA_PATH}` hint.  I believe these are both part of the osmesa installation.